tests/conftest.py

```diff
@@ -299,8 +299,15 @@ def mechanical_pool():
 
 
 def pytest_addoption(parser):
+    mechanical_path = atp.get_mechanical_path(False)
+
+    if mechanical_path == None:
+        parser.addoption("--ansys-version", default="232")
+    else:
+        mechanical_version = atp.version_from_path("mechanical", mechanical_path)
```

Comment:
What do we want it to do if there are multiple versions installed? just take the default from atp, or default to a specific version like 232 where the tests are known to be stable?

Reply:
Can we take the default from atp as long as it finds the mechanical_path above? I have multiple versions installed on my computer and atp defaulted to 232

Reply:
IMO, we should let atp decide - whatever the latest version is available, that's the one we take. atp provides them back in an orderly manner from latest to oldest, if I remember properly. That should be the behavior.